generate-type-definitions.js

```diff
@@ -3,8 +3,12 @@
 const { execSync } = require('child_process');
 const { readFileSync, writeFileSync } = require('fs');
 
-execSync('tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly');
-
+try {
+	execSync('tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly');
+} catch (err) {
+	console.error(err.stderr.toString());
+	throw err;
+}
```

Comment:
Without this PR, difficult to understand the error reason because it's like this.

>   stdout: Buffer(99) [Uint8Array] [
    115, 114,  99,  47, 114, 117, 110, 116, 105, 109, 101,  47,
    116, 114,  97, 110, 115, 105, 116, 105, 111, 110,  47, 105,
    110, 100, 101, 120,  46, 116, 115,  40,  50,  48,  49,  44,
     57,  41,  58,  32, 101, 114, 114, 111, 114,  32,  84,  83,
     54,  49,  51,  51,  58,  32,  39, 116,  39,  32, 105, 115,
     32, 100, 101,  99, 108,  97, 114, 101, 100,  32,  98, 117,
    116,  32, 105, 116, 115,  32, 118,  97, 108, 117, 101,  32,
    105, 115,  32, 110, 101, 118, 101, 114,  32, 114, 101,  97,
    100,  46,  10
  ]

With this PR, the error message is like this.

> src/runtime/transition/index.ts(201,9): error TS6133: 't' is declared but its value is never read.